### PR TITLE
CASM-2670: Refactor to use patched CSM images

### PR DESCRIPTION
**Pull request by wiltonpaulo**
CASM-2670: Refactor to use patched CSM images

### CODE REVIEW

File: Jenkinsfile.github [hunk 1]
```diff
@@ -12,26 +12,24 @@ pipeline {
 
     environment {
         NAME = "cray-precache-images"
-        DESCRIPTION = "Deploys a tool to cache some images on the system."
+        DESCRIPTION = "Helm chart to deploy node-problem-detector to Cray system."
```
Comment (brantk-hpe) [inline Jenkinsfile.github:15]: This is deploying cray-precache-images, not node-problem-detector

**Review by bklei — APPROVED** [review]

**Review by kimjensen-hpe — APPROVED** [review]

File: charts/cray-precache-images/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+name: cray-precache-images
+version: 0.4.0
+description: Cache nexus and other images across worker nodes
+keywords:
+  - precache-images
+  - precache
+  - images
+maintainers:
+  - name: bklei
+  - name: brantk-hpe
```
Comment (brantk-hpe) [inline charts/cray-precache-images/Chart.yaml:14]: I've had nothing to do with this chart.

File: charts/cray-precache-images/values.yaml [hunk 3]
```diff
@@ -16,5 +14,9 @@ cacheRefreshSeconds: "86400"
 nameOverride: ""
 fullnameOverride: ""
 podAnnotations: {}
-imagesHost: "dtr.dev.cray.com"
-alpineImage: baseos/alpine:3.12
+
+image:
+  alpine:
```
Comment (brantk-hpe) [inline charts/cray-precache-images/values.yaml:19]: this doesn't match what we're told to use in https://connect.us.cray.com/jira/browse/CASM-2670 (See Image section)

**Review by brantk-hpe — APPROVED** [review]

File: charts/cray-precache-images/Chart.yaml (hunk 2 above)
Comment (zcrisler) [reply charts/cray-precache-images/Chart.yaml]: Removed you.

File: Jenkinsfile.github (hunk 1 above)
Comment (zcrisler) [reply Jenkinsfile.github]: Corrected.

File: charts/cray-precache-images/values.yaml (hunk 3 above)
Comment (zcrisler) [reply charts/cray-precache-images/values.yaml:19]: What bit are you referring to? I hope I haven't given instructions that conflict with:

> As a general rule, do not reference images using remote, mirror, or proxy image repositories. Even though we maintain remotes for various registries in artifactory.algol60.net, these should only be used to enable container runtime efficiency.

Comment (zcrisler) [reply charts/cray-precache-images/values.yaml:19]: Ah, nevermind I see -- misread the line your comment applied to. Fixing...

Comment (zcrisler) [reply charts/cray-precache-images/values.yaml]: Changed from `images.alpine` to simply `images`.

**Pull request merged**